src/authentication/views/registration-flow/r4-manual-registration.tsx

```diff
@@ -21,7 +21,7 @@ import {
 
 import { APP_PATH } from '../../../constants';
 import { toastService } from '../../../shared/services';
```

Comment:
`import { toastService, ZendeskService } from '../../../shared/services';`

Reply:
Zou het trouwens ook zendeskService* noemen voor consistency.

Reply:
toastService zou eigenlijk ToastService moeten noemen, aangezien die beter een class geweest was met statische functies.

ZendeskService is een class en een class moet met uppercase camelcase volgens tslint.

Reply:
Moeten we idd is (in een aparte PR) gelijk trekken 👍

Reply:
https://github.com/viaacode/avo2-client/pull/426